### DISCUSSION TITLE
Copilot not working on neovim

### DISCUSSION THREAD

**Discussion by stopachka** [community/community#41869] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Hey team. I loved copilot on vscode, and wanted to set it up in neovim. But I am having trouble. Right now I get the error: 

```
Cannot source a directory: "/Users/stopa/.config/nvim/autoload/copilot.vim"
Cannot source a directory: "/Users/stopa/.config/nvim/autoload/copilot.vim"
Error detected while processing /Users/stopa/.config/nvim/autoload/copilot.vim/plugin/copilot.vim:
line   87:
E117: Unknown function: copilot#Init
```

I've installed it through vim plug: 

```
  Plug 'github/copilot.vim', {'branch': 'release'}
```

Looking around at bug reports, I saw I needed to have an older version of node. So I set that too with fnm: 

```
node --version
v17.0.1
```

I also tried setting the path manually in case nvim doesn't see it: 

```
let g:copilot_node_command = "/var/folders/sz/rnrd6cv509x2t8nbm6k4mzsh0000gn/T/fnm-shell-5104420/bin/node"
```

But alas I get this bug. 

What could I do to debug further?

**Comment by Akash1134** [comment]
Hi @stopachka We're sorry that you're facing issues using Copilot with Neovim. Please note that NeoVim had older versions of the copilot.vim extensions are no longer supported. Please upgrade to the latest version of your copilot.vim extension. Thanks!

**Comment by tpope** [comment, marked as answer]
You have a second copy of the plugin installed in the wrong place. Delete it:

```
rm -rf /Users/stopa/.config/nvim/autoload/copilot.vim
```

**Reply by stopachka** [reply]
Hey @tpope, thanks for the response! 

When I PlugInstall, this is the directory that copilot gets put into. Is it supposed to go somewhere else? (If I delete that folder, it's as though copilot was never installed)

**Reply by tpope** [reply]
From the [vim-plug](https://github.com/junegunn/vim-plug) README:

> ```
> " The default plugin directory will be as follows:
> "   - Vim (Linux/macOS): '~/.vim/plugged'
> "   - Vim (Windows): '~/vimfiles/plugged'
> "   - Neovim (Linux/macOS/Windows): stdpath('data') . '/plugged'
> " You can specify a custom plugin directory by passing it as the argument
> "   - e.g. `call plug#begin('~/.vim/plugged')`
> "   - Avoid using standard Vim directory names like 'plugin'
> ```

The relevant default for you should be `~/.local/share/nvim/plugged`. Make sure you're not calling `plug#begin()` with an argument.

**Reply by stopachka** [reply]
Ah! Okay I was calling `plug#begin()` with an argument. Removed and things are working honky dory. Thanks @tpope

**Comment by bhingston-va** [comment]
Doesn't answer this direct problem. I also ran into a similar issue (not same error but copilot stop working). Note it worked before. I found it had to do with wilder (https://vimawesome.com/plugin/wilder-nvim, https://github.com/gelguy/wilder.nvim).

Removing also solved it for me. I acknowledge this is likely a symptom to how I have neovim set up and some of the configuration. I wasn't able to remove the argument in `plug#begin()`. When I have more time I will likely dive deeper to understand the root of my issue to give better solution other than remove a plugin.
